__docs__/wonder-blocks-form/text-area.stories.tsx

```diff
@@ -158,14 +162,36 @@ export const ReadOnly: StoryComponentType = {
     },
 };
 
+/**
+ * If the `error` prop is set to true, the TextArea will have error styling and
+ * `aria-invalid` set to `true`.
+ *
+ * This is useful for scenarios where we want to show an error on a
+ * specific field after a form is submitted (server validation).
+ *
+ * Note: The `required` and `validate` props can also put the TextArea in an
+ * error state.
+ */
+export const Error: StoryComponentType = {
+    render: ControlledTextArea,
+    args: {
+        value: "With error",
+        error: true,
+    },
+};
+
 /**
  * If the textarea fails validation, `TextArea` will have error styling.
+ *
+ * This is useful for scenarios where we want to show errors while a
+ * user is filling out a form (client validation).
+ *
  * Note that we will internally set the correct `aria-invalid` attribute to the
  * `textarea` element:
- * - `aria-invalid="true"` if there is an error message.
- * - `aria-invalid="false"` if there is no error message.
+ * - `aria-invalid="true"` if there is an error.
+ * - `aria-invalid="false"` if there is no error.
  */
-export const Error: StoryComponentType = {
+export const ErrorFromValidation: StoryComponentType = {
```

Comment:
**question:** Do you think we should disable snapshots for this story? or maybe the `Error` one, or both? I'm not sure if we are testing different visual things and/if this is already covered in some way by the "All variants" stories.

Reply:
Good idea! The "All variants" stories covers the error state already, I've disabled snapshots for the error related stories for TextField and TextArea!